md/admin-application-resources-list.md

```diff
@@ -0,0 +1,106 @@
+# Resources list in the Bonita Administrator Application
+
+This page defines the term "Resources" and explains what a user with the _Administrator_ profile in the Bonita Living Application can see and do about resources.  
+
+_Administrators_ can view a list of the resources filtered by type, search a resource by its name, install new resources, update existing resources, or delete resources.  
+
+Here is a view of the page:
+![Administrator Resources](images/UI2021.1/admin-application-resources-list.png)<!--{.img-responsive}-->
+
+
+## Resource definition
+
+A resource is deployed in the Bonita Administrator Application to create the User Interface of a Living Application: it is a page, a layout, a theme, a REST API extension, but also a form that will be used by several processes so it is better to store it at platform level than within the process .bar file.  
```

Comment:
```suggestion
A resource is deployed in the Bonita Administrator Application to create the User Interface of a Living Application: it can be a page, a layout, a theme, a REST API extension, or a form that will be used by several processes so it is better to store it at platform level than within the process .bar file.  
```